pkg/skaffold/initializer/init.go

```diff
@@ -54,6 +54,18 @@ type Initializer interface {
 	GetImages() []string
 }
 
+// InitBuilder represents a builder that can be chosen by skaffold init.
+type InitBuilder interface {
+	// GetPrompt returns the initBuilder's string representation, used when prompting the user to choose a builder.
+	GetPrompt() string
+	// GetArtifact returns the Artifact used to generate the Build Config.
```

Comment:
`GetArtifact creates an Artifact to be included in the generated Build Config` ?